api/explorer.go

```diff
@@ -188,13 +194,18 @@ func (api *API) buildExplorerBlock(height types.BlockHeight, block types.Block)
 		panic("incorrect request to buildExplorerBlock - block does not exist")
 	}
 
-	return ExplorerBlock{
+	b := ExplorerBlock{
 		MinerPayoutIDs: mpoids,
 		Transactions:   etxns,
 		RawBlock:       block,
+		HexBlockID:     hex.EncodeToString(encoding.Marshal(block.ID())),
```

Comment:
As mentioned above this can probably just be a `type.BlockID` instead of having to encode yourself. That means it would be `BlockID: block.ID(),`